frontend/src/components/tabs/MetricsTab.test.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2021 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Artifact, ArtifactType, Execution, Value } from '@kubeflow/frontend';
+import { render, waitFor } from '@testing-library/react';
+import { Struct } from 'google-protobuf/google/protobuf/struct_pb';
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { BrowserRouter } from 'react-router-dom';
+import { MetricsTab } from './MetricsTab';
+
+describe('MetricsTab', () => {
+  const mlmdUtils = require('src/lib/MlmdUtils');
```

Comment:
nit: you can do this instead

```
import * as mlmdUtils from 'src/lib/MlmdUtils'
jest.spyOn(mlmdUtils, 'xxxx')
```

this avoids using require

Reply:
Good point! Updated.